README.md

```diff
@@ -363,16 +363,19 @@ dependencies {
   // core coroutines
   implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.4.3")
   implementation("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.4.3")
+  
+  // a BOM ensures that all artifacts used from the library are of the same version
+  implementation(platform("com.rickbusarow.dispatch:dispatch-bom:1.0.0-RC01"))
```

Comment:
Nit: You might want to tell the people that they have to add the platform tag to other configurations as well, i.e. `testImplementation` and `androidTestImplementation`. This alone is of course not very useful and saves no writing, however, people can set up their own "production" or "test" Java platform, import your BOM there and reference this here, so all their production / test dependencies are refererenced in a unified matter.

Reply:
Oh wait, `testImplementation` actually inherits `implementation`, ignore me... I was thinking about `kapt` and others for a moment. All good on your side :)

Reply:
:) Thanks.  I plan to update _all_ of this documentation before the 1.0 release.  I know it's pretty lacking.